lib/weather_report.ex

```diff
@@ -73,6 +73,14 @@ defmodule WeatherReport do
             end)
   end
 
+  @doc """
+  Refresh cached stations.
+  """
+  @spec refresh_stations() :: :ok
+  def refresh_stations() do
+    :ok = GenServer.cast(StationRegistry, :refresh_list)
+  end
```

Comment:
Made this so I didn't have to end an iex session and restart it just to get the stations to update. Was useful so kept it in.